Source/Generator/_Editor/MeasurementService Editor.lvlib

```diff
@@ -4,6 +4,28 @@
 	<Property Name="NI.Lib.SourceVersion" Type="Int">536903680</Property>
 	<Property Name="NI.Lib.Version" Type="Str">1.0.0.0</Property>
 	<Property Name="NI.LV.All.SourceOnly" Type="Bool">true</Property>
+	<Item Name="RenameRefactor" Type="Folder">
```

Comment:
It might be nice to add a Readme. I have no idea how I'd use this utility. What VIs do I run? In what order? What does it do?

Reply:
On a similar note, if there is a single VI that you run, maybe you can somehow "promote" that as the main VI.

Reply:
If we make this public, that will resolve itself, but I get your meaning. I could also put this in a library with a single public method.

Reply:
Added README.md to the folder.